src/adapters/YieldAdapter.sol

```diff
@@ -20,14 +20,47 @@ contract YieldAdapter is IAdapter {
 
     address public redeemer;
 
+    // @notice returns the address of the underlying token for the PT
+    // @param pt The address of the PT
     function underlying(address pt) public view returns (address) {
         return address(IYield(pt).base());
     }
 
+    // @notice returns the maturity of the underlying token for the PT
+    // @param pt The address of the PT
     function maturity(address pt) public view returns (uint256) {
         return IYield(pt).maturity();
     }
 
+    // @notice lendABI "returns" the arguments required in the bytes `d` for the lend function
+    // @returns underlying_ The address of the underlying token
+    // @returns maturity The maturity of the underlying token
+    // @returns minimum The minimum amount of the PTs to receive when spending (amount - fee)
+    // @returns pool The address of the pool to lend to (buy PTs from)
+    function lendABI(
+    ) public pure returns (
+        address underlying_,
```

Comment:
i think we can remove the dangling underscore from `underlying_` here...

same in the `redeemABI` method

Reply:
there are conflicts with the `underlying` method on the adapter itself when you do -- they dont impact execution but solidity does yell at you for it 🤷